seqr/views/utils/export_utils_tests.py

```diff
@@ -43,21 +43,21 @@ def test_export_table(self):
     def test_export_multiple_files(self, mock_zip):
         mock_zip_content = {}
         def mock_write_zip(file, content):
-            mock_zip_content[file] = unicode(content)
+            mock_zip_content[file] = content.decode('utf-8')
         mock_zip.return_value.__enter__.return_value.writestr.side_effect = mock_write_zip
 
         header1 = ['col1', 'col2']
         header2 = ['col1']
         header3 = ['col2', 'col3', 'col1']
-        rows = [{'col2': 'row1_v2', 'col1': 'row1_v1\xe2'}, {'col1': 'row2_v1'}]
+        rows = [{'col2': 'row1_v2', 'col1': u'row1_v1\xe2'}, {'col1': 'row2_v1'}]
 
         # test csv format
-        response = export_multiple_files([['file1', header1, rows], ['file2', header2, rows]], 'zipfile')
+        response = export_multiple_files([[u'file1\xe3', header1, rows], ['file2', header2, rows]], 'zipfile')
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.get('content-disposition'), 'attachment; filename="zipfile.zip"')
         self.assertDictEqual(mock_zip_content, {
-            'file1.csv': 'col1,col2\nrow1_v1,row1_v2\nrow2_v1,',
-            'file2.csv': 'col1\nrow1_v1\nrow2_v1',
+            u'file1\xe3.csv': u'col1,col2\nrow1_v1\xe2,row1_v2\nrow2_v1,',
```

Comment:
instead of prefixing all the test file strings with `u`, why not just use the future unicode import in this file?

Reply:
I didn't convert this file to Python2 and 3 compatible yet. But I AM able to use the unicode_literals to covert all the str literals to unicode before doing other converting.

Reply:
I'm going to convert the whole file to be Py2/3 compatible after import unicode_literals. The only difference is to replace `self.assertRaisesRegexp()` with `self.assertRaisesRegex()`.